tests/ui/perfherder/alerts-view/select_alerts_dropdown_test.jsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { render, cleanup, waitFor, fireEvent } from '@testing-library/react';
+
+import testAlertSummaries from '../../mock/alert_summaries';
+import SelectAlertsDropdown from '../../../../ui/perfherder/alerts/SelectAlertsDropdown';
+import { alertStatusMap } from '../../../../ui/perfherder/constants';
+import { getStatus } from '../../../../ui/perfherder/helpers';
+
+const testUser = ({ isLoggedIn, isStaff }) => ({
+  username: 'mozilla-ldap/test_user@mozilla.com',
+  isLoggedIn,
+  isStaff,
+  email: 'test_user@mozilla.com',
+});
+const testAlertSummary = testAlertSummaries[0];
```

Comment:
This summary only has untriaged alerts. We should make sure we have multiple alerts with multiple statuses. I think `untriaged`, `invalid` & `acknowledged` would suffice.